src/actions/guides/getting-started/installing.cr

```diff
@@ -29,51 +27,127 @@ class Guides::GettingStarted::Installing < GuideAction
 
-    ### Debian and Fedora dependencies
+    ### Ensure Postgres CLI tools installed
+
+    If you're using [Postgres.app](https://postgresapp.com) on macOS make sure
+    [Postgres CLI tools](https://postgresapp.com/documentation/cli-tools.html) are installed
+
+    ```plain
+    sudo mkdir -p /etc/paths.d &&
+      echo /Applications/Postgres.app/Contents/Versions/latest/bin | sudo tee /etc/paths.d/postgresapp
+    ```
+
+    There are other installation methods available in [Postgres CLI tools docs](https://postgresapp.com/documentation/cli-tools.html)
+
+    ## Linux dependencies
+
+    Debian (Ubuntu should be similar)
+
+    ```plain
+    apt-get install libc6-dev libevent-dev libpcre2-dev libpng-dev libssl1.0-dev libyaml-dev zlib1g-dev
+    ```
 
-    * Debian (Ubuntu should be similar): run `apt-get install libc6-dev libevent-dev libpcre2-dev libpng-dev libssl1.0-dev libyaml-dev zlib1g-dev`
-    * Fedora (28): run `dnf install glibc-devel libevent-devel pcre2-devel openssl-devel libyaml-devel zlib-devel libpng-devel`. libpng-devel is for Laravel Mix
+    Fedora (28)
 
-    ### For building assets (skip if building APIs)
+    ```plain
+    dnf install glibc-devel libevent-devel pcre2-devel openssl-devel libyaml-devel zlib-devel libpng-devel
+    ```
+
+    ## Chromedriver (optional)
+
+    > You can skip this if you only plan to build APIs.
+
+    If you want to test your frontend using
+    [LuckyFlow](#{Guides::Frontend::Testing.path}) you will need
+    Chromedriver, see the [Testing HTML and
+    Interactivity](#{Guides::Frontend::Testing.path}) guide for details and installation.
+
+    ## Node and Yarn (optional)
+
+    > You can skip this if you only plan to build APIs.
 
     * [Node](https://nodejs.org/en/download/). Requires at least v6
     * [Yarn](https://yarnpkg.com/lang/en/docs/install/)
 
-    ### Dependencies for browser tests
+    ## Install Lucky CLI on macOS
+
+    Once the required dependencies above are installed, set up Lucky for your system.
+
+    ### Install Homebrew
+
+    Installation instructions from the [Homebrew website](https://brew.sh/)
+
+    ```plain
+    sudo mkdir -p /etc/paths.d &&
+    echo /Applications/Postgres.app/Contents/Versions/latest/bin | sudo tee /etc/paths.d/postgresapp
+    ```
+
+    ### Add the Lucky tap to Homebrew
+
+    ```plain
+    brew tap luckyframework/homebrew-lucky
+    ```
+
+    ### Install the Lucky CLI with Homebrew
+
+    ```plain
+    brew install lucky
+    ```
+
+    ## Install Lucky CLI on Linux
+
+    ### Clone the CLI repo
+
+    ```plain
+    git clone https://github.com/luckyframework/lucky_cli
+    ```
+
+    ### Change into the newly clone directory
+
+    ```plain
+    cd lucky_cli
+    ```
+
+    ### Check out the latest released version
+
+    ```plain
+    git checkout #{LuckyCliVersion.current_tag}
+    ```
+
+    > See a list of [released version](https://github.com/luckyframework/lucky_cli/releases)
```

Comment:
Not sure why, but reading down this, this part threw me off. I thought that was the end of the instructions, and I thought "what about building the binary?". Then I went back and looked and went "oh, there's more".. Maybe just make this text inside of a side note?

Reply:
Yeah good idea! It's not that important so can be a regular text note